packages/language-server/test/logger-spec.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import { getLogger, getLogLevel, setLogLevel } from "../src/logger";
+import { LogLevel } from "@vscode-logging/logger";
+
+describe("the Language Server Logger", () => {
+  let errorSpy;
+  let sandbox;
+
+  before(() => {
+    sandbox = sinon.createSandbox();
```

Comment:
I think you don't have to create a sandbox anymore since sinon has a default sandbox (you can call `sinon.spy`, `sinon.restore` etc instead of using a specific sandbox)

Reply:
Thanks